doc/rst/source/cookbook/features.rst

```diff
@@ -2402,17 +2402,18 @@ the data may need translation and scaling prior to use. Therefore, all
 GMT programs that read or write grid files will decode the given
 filename as follows:
 
-name[=\ *ID*][**+s**\ *scale*][**+o**\ *offset*][**+n**\ *invalid*]
+name[=\ *ID*][**+d**\ *divisor*][**+n**\ *invalid*][**+o**\ *offset*][**+s**\ *scale*]
```

Comment:
Also needs to be updated in explain_grd_inout.rst_

Reply:
Done, as well as in grdconvert.rst which has more of the same.